pkg/plugins/golang/v4/scaffolds/internal/templates/dockerfile.go

```diff
@@ -39,7 +39,7 @@ func (f *Dockerfile) SetTemplateDefaults() error {
 }
 
 const dockerfileTemplate = `# Build the manager binary
-FROM golang:1.20 as builder
+FROM golang:1.20 AS builder
```

Comment:
Could we have a PR for each fix as well?
Could you please open another PR for this one?

Reply:
sure, will update this one and create the other one in a few

Reply:
done